Buildings/Experimental/DHC/CentralPlants/BaseClasses/PartialPlant.mo

```diff
@@ -129,6 +120,16 @@ partial model PartialPlant
     annotation (
       Placement(transformation(extent={{300,100},{340,140}}),
         iconTransformation(extent={{300,80},{380,160}})));
+protected
+  final parameter Boolean have_heatingPort=typ <> Buildings.Experimental.DHC.Types.DistrictSystemType.Cooling and
+  typ <> Buildings.Experimental.DHC.Types.DistrictSystemType.CombinedGeneration5
+  "Boolean flag to enable fluid connectors for heating service line";
+  final parameter Boolean have_coolingPort=typ == Buildings.Experimental.DHC.Types.DistrictSystemType.CombinedGeneration1 or
+  typ == Buildings.Experimental.DHC.Types.DistrictSystemType.CombinedGeneration2to4 or
+  typ == Buildings.Experimental.DHC.Types.DistrictSystemType.Cooling
+  "Boolean flag to enable fluid connectors for cooling service line";
+  final parameter Boolean have_ambientPort=typ == Buildings.Experimental.DHC.Types.DistrictSystemType.CombinedGeneration5
+  "Boolean flag to enable fluid connector for ambient water service line";
```

Comment:
Is there a reason for deviating from the naming convention where 3 characters are used, which is also what is used for example in `Buildings.Experimental.DHC.EnergyTransferStations.Combined.Generation5.ChillerBorefield`.
Wouldn't a consistent nomenclature use `have_hea`, `have_coo` and `have_serAmb`, or is there a compelling reason for this departure of what is used in the other classes of the DHC package?

Reply:
Thanks for pointing this out! It is indeed better to unify these names to follow the convention to be consistent with other related cases, I will revise these!